hetnet/neo4j/deployment/install_ssl.sh

```diff
@@ -25,22 +25,26 @@ sudo certbot certonly \
 # Create "sync-neo4j-ssl.sh" dynamically and run it:
 cat > ./sync-neo4j-ssl.sh << EOF
 #!/bin/bash
-# Certbot post-renewal-hook script that synchronizes SSL certificates for neo4j
+# Certbot deploy-renewal-hook script, which synchronizes SSL certificates for neo4j.
+# This script will be executed ONLY WHEN certificate is renrewed successfully.
 
 # Use 'cp --dereference' to emphasize that we are copying the actual files.
 cp --dereference --force /etc/letsencrypt/live/$SSL_DOMAIN/fullchain.pem /home/ubuntu/ssl/neo4j.cert
 cp --dereference --force /etc/letsencrypt/live/$SSL_DOMAIN/privkey.pem   /home/ubuntu/ssl/neo4j.key
-
-# If hetionet-container is running now, restart it to make the new certificates effective.
-if [ \`docker ps --quiet --filter name=hetionet-container\` ]; then
-    echo -n "Restarting "; docker restart hetionet-container
-fi
 EOF
 
 mkdir -p /home/ubuntu/ssl/
 chmod +x ./sync-neo4j-ssl.sh
 sudo ./sync-neo4j-ssl.sh
 
-# Add post-renewal-hook, see:
+# If hetionet-container is running now, restart it to make the new certificates effective.
+if [ -n $(docker ps --quiet --filter name=hetionet-container) ]; then
+    echo -n "Restarting "
+    docker restart hetionet-container
+fi
```

Comment:
So the command above is just for the initial installation of SSH and the hooks are for future renewals?

Reply:
Yes.